@navikt/ds-react/src/form/checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Checkbox, CheckboxGroup } from ".";
+
+test("checkbox group chains onChange calls", () => {
+  const onGroupChange = jest.fn();
+  const onChange = jest.fn();
+  const value = "value";
+  const label = "label";
+
+  render(
+    <CheckboxGroup legend="legend" onChange={onGroupChange}>
+      <Checkbox onChange={onChange} value={value}>
+        {label}
+      </Checkbox>
+    </CheckboxGroup>
+  );
+
+  userEvent.click(screen.getByLabelText(label));
+
+  expect(onGroupChange).toBeCalledTimes(1);
+  expect(onGroupChange).toBeCalledWith([value]);
+  expect(onChange).toBeCalledTimes(1);
+  expect(onChange.mock.calls[0][0].target.checked).toBe(true);
```

Comment:
Vil det være bedre å bruke vanlig ref for linje 25? Eller kan det regnes som en dårlig test da siden ref evt kan feile?

Reply:
Kan du forklare hva du mener med "å bruke vanlig ref"?

Reply:
```jsx
const chkRef = useRef()

<Checkbox ref={chkRef} onChange={onChange} value={value}>
	{label}
</Checkbox>

expect(chkRef.current.checked).toBe(true);
```

Reply:
Ikke dumt. Her er et annet alternativ.
```ts
const checkbox = screen.getByLabelText(label) as HTMLInputElement;
userEvent.click(checkbox);
expect(checkbox.checked).toBe(true);
```

Reply:
Det jeg liker med testen sånn som jeg har skrevet den er at den sjekker formatet på `onChange` sitt første argument.

Hva tenker du er fordeler og ulemper med de tre løsningene?

Reply:
Første tanke var bare at det var litt kryptisk hva `onChange.mock.calls[0][0].target` faktisk gjorde. Men ser hvordan denne måte kan være god siden man sikrer at man sjekker det elementet som faktisk er endret.

Reply:
Jeg lagde en funksjon (`firstArgumentOfFirstCall`) for å gjøre testen enklere å forstå. Bra innspill!